helm/values.yaml

```diff
@@ -0,0 +1,360 @@
+hypertraceAlertEngineConfig:
+  name: hypertrace-alert-engine-config
+  replicaCount: 1
+  maxUnavailable: 0
+  image:
+    repository: hypertrace/hypertrace-alert-engine
+    pullPolicy: IfNotPresent
+    tagOverride: ""
+  imagePullSecrets: [ ]
+  nodeLabels: { }
+  containerAdminPort: 8099
+  javaOpts: "-XX:InitialRAMPercentage=50.0 -XX:MaxRAMPercentage=75.0 -XX:MaxDirectMemorySize=128M"
+  livenessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 15
+    failureThreshold: 10
+  readinessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 15
+    failureThreshold: 10
+  deploymentLabels:
+    app: hypertrace-alert-engine
+  podLabels:
+    app: hypertrace-alert-engine
+  podAnnotations: { }
+  deploymentSelectorMatchLabels:
+    app: hypertrace-alert-engine
+  serviceSelectorLabels:
+    app: hypertrace-alert-engine
+  resources:
+    requests:
+      cpu: 0.1
+      memory: 1024Mi
+    limits:
+      cpu: 0.2
+      memory: 1024Mi
+  logConfig:
+    name: hypertrace-alert-engine-log-appender-config
+    monitorInterval: 30
+    rootLogger:
+      level: INFO
+    appender:
+      rolling:
+        enabled: false
+
+  queryService:
+    host: query-service
+    port: 9012
+  attributeService:
+    host: attribute-service
+    port: 9012
+  notificationChannelsConfig:
+    name: hypertrace-alert-engine-notification-channels-config
+    channels:
+      [
```

Comment:
These examples shouldn't be in the values

Reply:
they can be kept empty and finally be overridden by the user?

Reply:
These are the defaults right? generally, these would be created by users at runtime, not as server config.

Reply:
these are default file based rules (have to be passed through helm override)